# generated by patch-package 6.4.10
#
# command:
#   npx patch-package parcel-transformer-svelte3-plus
#
# declared package:
#   parcel-transformer-svelte3-plus: 0.2.7
#
diff --git a/node_modules/parcel-transformer-svelte3-plus/index.js b/node_modules/parcel-transformer-svelte3-plus/index.js
index 3ef6b92..c4d1bbc 100644
--- a/node_modules/parcel-transformer-svelte3-plus/index.js
+++ b/node_modules/parcel-transformer-svelte3-plus/index.js
@@ -52,15 +52,18 @@ module.exports = new Transformer({
         ['.svelterc', 'svelte.config.js'],
         {packageKey: 'svelte'},
     );
+
+  
     let contents = {};
     if (conf) {
       contents = conf.contents;
       if (typeof contents !== 'object') {
-        throw new ThrowableDiagnostic({
-          diagnostic: {
-            message: 'Svelte config should be an object.',
-          },
-        });
+        console.log(contents)
+        // throw new ThrowableDiagnostic({
+        //   diagnostic: {
+        //     message: 'Svelte config should be an object.',
+        //   },
+        // });
       }
       if (conf.filePath.endsWith('.js')) {
         config.invalidateOnStartup();
